packages/ods/src/components/quantity/src/controller/ods-quantity.ts

```diff
@@ -0,0 +1,17 @@
+function isAddButtonDisabled(isDisabled: boolean, value: number | null, max?: number): boolean {
+  return isDisabled || (max !== undefined && max === value);
```

Comment:
this enabled button if value is bigger than the max (which is doable by typing directly the value in the input)
`return isDisabled || (max !== undefined && value >= max);`

Reply:
Add tests accordingly

Reply:
It's because of the input value, it's like type text in the input, the event `onInput` is not trigger and the input is not valid